### DISCUSSION TITLE
Re-run workflows and jobs

### DISCUSSION THREAD

**Discussion by efirdman** [community/community#50350] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

https://docs.github.com/en/actions/managing-workflow-runs/re-running-workflows-and-jobs

Is the re-running workflow and jobs still available?

Specifically for **successful jobs**?

I cannot locate the button to start the re-run

**Comment by Mohamed-SayedAlAhl** [comment]
Yes, the ability to re-run workflows and jobs is still available in GitHub Actions. To re-run a successful job, you can click on the job in the "Jobs" tab of the workflow run and then click the "Re-run job" button in the top-right corner of the job details page. To re-run the entire workflow, you can click the "Re-run jobs" button in the top-right corner of the workflow run page. 

Note that you can only re-run successful jobs or the entire workflow if there are no currently running or pending jobs in the workflow. If there are any running or pending jobs, the "Re-run" button will be disabled until they have all completed.
